clang/test/CodeGenSYCL/kernel-param-member-acc-array.cpp

```diff
@@ -0,0 +1,84 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -I %S/Inputs -fsycl-int-header=%t.h -triple spir64-unknown-unknown-sycldevice -disable-llvm-passes -emit-llvm %s -o - | FileCheck --dump-input-on-failure %s
```

Comment:
FileCheck doc says that `--dump-input-on-failure` option is deprecated. Also, I'm not sure that it is needed here at all.

Reply:
OK